src/utils/validations.js

```diff
@@ -110,3 +112,38 @@ export const validateVmwareURL = value => {
 */
   return null;
 };
+
+// <protocol>://<host>:<port>
+//
+// "protocol" is optional, defaults to "ipmi"
+// "host" is a hostname or an IP address
+// "port" is optional
+
+export const validateBmcURL = value => {
+  if (!value) {
+    return getValidationObject(EMPTY_ERROR);
+  }
+
+  const hasProtocol = value.includes('://');
+  const hasPort = value.match(':(?!/)');
+
+  if (hasProtocol) {
+    const startsWithAllowedProtocol = startsWith(value, 'ipmi') || startsWith(value, 'idrac');
```

Comment:
nitpick: could be also done this way
 
```js
if(hasProtocol && !['impi', 'idrac'].find(allowedProtocol => startsWith(value, allowedProtocol)){
  return getValidationObject(BMC_PROTOCOL_ERROR);
}
```

and `['impi', 'idrac']` could be  extracted to a constant.

Reply:
Done